flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/DatabaseSync.java

```diff
@@ -72,16 +76,19 @@ public abstract class DatabaseSync {
 
     public void create(StreamExecutionEnvironment env, String database, Configuration config,
                        String tablePrefix, String tableSuffix, String includingTables,
-                       String excludingTables, boolean ignoreDefaultValue, Configuration sinkConfig,
+                       String excludingTables,String multiToOneOrigin,String multiToOneTarget, boolean ignoreDefaultValue, Configuration sinkConfig,
             Map<String, String> tableConfig, boolean createTableOnly, boolean useNewSchemaChange) {
         this.env = env;
         this.config = config;
         this.database = database;
-        this.converter = new TableNameConverter(tablePrefix, tableSuffix);
         this.includingTables = includingTables;
         this.excludingTables = excludingTables;
+        this.multiToOneOrigin = multiToOneOrigin;
+        this.multiToOneTarget = multiToOneTarget;
         this.includingPattern = includingTables == null ? null : Pattern.compile(includingTables);
         this.excludingPattern = excludingTables == null ? null : Pattern.compile(excludingTables);
+        this.multiToOneRulesPattern=multiToOneRulesParser(multiToOneOrigin,multiToOneTarget);
```

Comment:
Please manually format the code, such as spaces on both sides of =, thank you.
ps: In the future, code check-style will be added to automatically help verify these.

Reply:
thanks,let me adjust it